db/python/layers/analysis.py

```diff
@@ -48,6 +50,7 @@ def __init__(self, connection: Connection):
 
         self.sampt = SampleTable(connection)
```

Comment:
I am not big fan of abbreviations like this, I know this is a bit of legacy, so no drama :-)
'self.sample_table' is clearer, the same for self.analysis_table, self.output_file_table

Reply:
Good suggestion, I've updated all those now :)